### PR TITLE
Polls November 15

### PR DESCRIPTION
**Pull request by Hipogeo**
Hey, @mkrorbrkr ! Here's the polls. Let me know if there's anything that needs to be changed :)

### CODE REVIEW

File: governance/polls/Increase the ilk.hole Parameter for Several Collaterals - November 15, 2021.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review 
+
+MKR Token Holders may vote in this poll to express support or opposition to increasing [ilk.hole](https://makerdao.world/en/learn/governance/param-local-liquidation-limit/) for the following vaults as follows:
+   * ETH-A: from current 40M to 65M.
+   * ETH-B: from current 25M to 30M.
+   * ETH-C: from current 30M to 35M.
+   * WBTC-A: from current 25M to 40M
+   * WSTETH-A: from current 3M to 7M
```
Comment (mkrorbrkr) [inline governance/polls/Increase the ilk.hole Parameter for Several Collaterals - November 15, 2021.md:32]: As before, would spell out "million" and indicate that this is a figure in DAI

See this poll for an example of how you might format this section: https://github.com/makerdao/community/blob/master/governance/polls/Adjusting%20Auction%20Parameters%20for%20ETH-A%2C%20ETH-C%2C%20and%20WBTC-A%20-%20September%2020%2C%202021.md

**Review by mkrorbrkr — CHANGES_REQUESTED** [review]
@Hipogeo just a few minor changes

**Review by mkrorbrkr — APPROVED** [review]
@prose11 @Hipogeo I think this is now good to go :)

File: governance/polls/Change of Covenants for P1-DROP (farmlands) - November 15, 2021.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: Change of Covenants for P1-DROP (farmlands) - November 15, 2021
+summary: Signal your support or opposition for increasing the acquisition price up to 2.85M for P1-DROP
+discussion_link: https://forum.makerdao.com/t/signal-request-p1-drop-change-of-covenants-for-p1-drop-farmlands/11236
+vote_type: Plurality Voting
+categories:
+   - Collateral
```
[the remark this answers is not in the document]
Comment (prose11) [reply governance/polls/Change of Covenants for P1-DROP (farmlands) - November 15, 2021.md:7]: Adding it in :)

**Review by prose11 — APPROVED** [review]
Looks good to me, I'll swap out "farmlands" for "People's Company Series 1" but given the signal I see why farmlands was put in

**Pull request merged**